application/models/artifact.py

```diff
@@ -1,66 +1,45 @@
-""" Elastic search artifact model wraps api into crud methods"""
-from ..schemas.artifact_schema import ArtifactSchema
-from .esmodel import ESModel
-
-
-class Artifact(ESModel):
-    """ Handles saving and searching """
-
-    index = "artifact"
-    schema = ArtifactSchema
-
-    def __init__(self, params):
-        """ Initializes the artifact """
-        super().__init__(params)
-        self.file_url = params.get("file_url", None)
-        self.tags = params.get("tags", [])
-        self.file_date = params.get("file_date", None)
-        self.team_id = params.get("team_id", None)
-        if "score" in params:
-            self.score = params["score"]
+from .elastic_artifact import ElasticArtifact
+from .neo_artifact import NeoArtifact
+from .neo_tag import NeoTag
+import uuid
+
+
+class Artifact:
```

Comment:
This is the wrapper for both the ElasticArtifact and the NeoArtifact. It isn't very pretty as I hope to get rid of it soon and do the syncing differently.